src/BuildingRegistry.Api.BackOffice.Abstractions/Building/Validators/ValidationErrors.cs

```diff
@@ -69,7 +69,7 @@ public static string BuildingInvalid(string buildingPuri)
             public const string BuildingUnitCannotBeRealized = "Deze actie is enkel toegestaan op gebouweenheden met status 'gepland'.";
             public const string BuildingUnitCannotBeNotRealized = "Deze actie is enkel toegestaan op gebouweenheden met status 'gepland'.";
             public const string BuildingUnitCannotBeCorrectedFromNotRealizedToPlanned = "Deze actie is enkel toegestaan op gebouweenheden met status 'nietGerealiseerd'.";
-            public const string BuildingUnitCannotBeCorrectedFromNotRealizedToPlannedBecauseOfInvalidBuildingStatus = "Deze actie is enkel toegestaan voor een 'gepland' of 'gerealiseerd' gebouw.";
+            public const string BuildingUnitCannotBeCorrectedFromNotRealizedToPlannedBecauseOfInvalidBuildingStatus = "Deze actie is enkel toegestaan binnen een gepland of gerealiseerd gebouw.";
```

Comment:
Doesn't there need to be ' between the statusses?

Reply:
Nope. Already checked that with Anaïs.